docs/user-guide/timeline.md

```diff
@@ -4,28 +4,34 @@
 The Timeline is a [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) tool for managing layers with a time dimension.
 It makes possible to observe the layers' evolution over time, to inspect the layer configuration at a specific time instant (or in a time range) and to view different layer configurations time by time dynamically through animations.<br>
 
```

Comment:
You forgot to preserve the original warining here:

https://github.com/geosolutions-it/MapStore2/pull/5170/files#r416469889